DESCRIPTION

```diff
@@ -20,7 +20,8 @@ Imports:
     git2r,
     rmarkdown,
     rstudioapi,
-    whisker
+    whisker,
```

Comment:
Can you please remove this?

Reply:
Done. I didn't catch that it was only used in the example. Cheers.